src/common_swagger_api/schema/apps.clj

```diff
@@ -543,6 +543,9 @@
           :disabled
           AppDisabledParam
 
+          (optional-key :version)
```

Comment:
Is this version key always going to be optional?  I figure once app versioning is complete, it might need to be required, but maybe for testing things out right now optional makes more sense.

Reply:
Good question, and I'm not sure.

This is only for app listings, where we didn't plan on showing the version in the UI, at least at first.
Version info should be required in any app detail responses, but I don't know about listing responses yet 🤔

Reply:
OK yeah, makes sense that it wouldn't be required in the listing, and either way if we're unsure then optional makes sense to me 👍